packages/core/src/App/Containers/CitizenshipModal/set-citizenship-form.jsx

```diff
@@ -53,12 +42,13 @@ const SetCitizenshipForm = ({
                 </React.Fragment>
             )}
         </Field>
-        {children}
+        <Text as='p' size='xxs' className='account-signup__subtext' color='less-prominent'>
```

Comment:
Same here

Reply:
There's no `children` already in this component.